Libraries/Image/RCTImageUtils.m

```diff
@@ -64,6 +64,7 @@ CGRect RCTTargetRect(CGSize sourceSize, CGSize destSize,
 
   switch (resizeMode) {
     case RCTResizeModeStretch:
+    case RCTResizeModeRepeat:
```

Comment:
I'm not 100% sure what these functions should return and what they are used for so maybe someone familiar with it could confirm that this is fine.